drizzlepac/wcs_functions.py

```diff
@@ -30,44 +29,43 @@
 class WCSMap:
     """ Sample class to demonstrate how to define a coordinate transformation
     """
-    def __init__(self,input,output,origin=1):
+    def __init__(self, input, output, origin=1):
         # Verify that we have valid WCS input objects
-        self.checkWCS(input,'Input')
-        self.checkWCS(output,'Output')
+        self.checkWCS(input, 'Input')
+        self.checkWCS(output, 'Output')
 
         self.input = input
         self.output = copy.deepcopy(output)
-        #self.output = output
 
         self.origin = origin
         self.shift = None
         self.rot = None
         self.scale = None
 
-    def checkWCS(self,obj,name):
+    def checkWCS(self, obj, name):
         try:
             assert isinstance(obj, pywcs.WCS)
         except AssertionError:
-            print(name +' object needs to be an instance or subclass of a PyWCS object.')
+            print(name + ' object needs to be an instance or subclass of a PyWCS object.')
```

Comment:
Maybe it is time to say bye-bye to `PyWCS`?

Reply:
Good idea...done.